java-core/builder-file/work-with-file.md

```diff
@@ -0,0 +1,32 @@
+### Common mistakes
+
+#### Divide program logic
+Large methods is very difficult to read and understand. So methods should be short, between 5-15 lines.
+And you should remember that one method should be responsible for one task.
```

Comment:
```suggestion
And it would be best if you remembered that one method should be responsible for one task.
```

Reply:
@boroda4436 
not sure it would be correct sentence. 
And it's important for you to remember that one method should be responsible for one task.